.github/workflows/ci.yml

```
@@ -658,11 +657,11 @@ jobs:
           DEPENDENCY_CACHE_KEY: ${{ needs.job_setup.outputs.dependency_cache_key }}
 
-      - name: Setup Playwright
-        uses: ./.github/actions/setup-playwright
-
-      - run: yarn nx run @tryghost/admin-x-settings:test:acceptance
-
+      - name: Run playwright tests
+        uses: ./.github/actions/docker-playwright
+        with:
+          test_command: "yarn nx run @tryghost/admin-x-settings:test:acceptance"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)